styles/styles.css

```diff
@@ -34,13 +36,12 @@
 
 .advantages {
     display: flex;
-    margin-left: 181px;
-    margin-bottom: 122px;
-
+    margin: 0 auto 122px 181px;
```

Comment:
you don`t actually need 181px here - remove it and you find out how actually margin auto works